neo/rawio/intanbinaryrawio.py

```diff
@@ -0,0 +1,438 @@
+"""
+Support for Intan's binary file mode which is turned on by setting the save
+option in Intan to 'One File Per Signal Type' or 'One File Per Channel'.
+The reader for this format is for rhd only since it uses the rhd header.
+Need to confirm if rhs can also use this save format.
+
+RHD supported 1.x, 2.x, and 3.0, 3.1, 3.2
+See:
+  * http://intantech.com/files/Intan_RHD2000_data_file_formats.pdf
+  * http://intantech.com/files/Intan_RHS2000_data_file_formats.pdf
+
+Author: Zach McKenzie, based on Samuel Garcia's IntanRawIO
+"""
+from pathlib import Path
+
+import numpy as np
+from collections import OrderedDict
+from packaging.version import Version as V
+
+from .baserawio import (
+    BaseRawIO,
+    _signal_channel_dtype,
+    _signal_stream_dtype,
+    _spike_channel_dtype,
+    _event_channel_dtype,
+    _common_sig_characteristics,
+)
+
+from .intanrawio import (
+    read_variable_header,
+    rhd_global_header_base,
+    rhd_global_header_part1,
+    rhd_global_header_v11,
+    rhd_global_header_v13,
+    rhd_global_header_v20,
+    rhd_global_header_final,
+    rhd_signal_group_header,
+    rhd_signal_channel_header,
+    stream_type_to_name,
```

Comment:
@h-mayorquin, taken from intanrawio here for the mapping. But I think you suggest be clearer and just retyping this mapping out in the file?